src/universal/components/Dashboard/dashTimestamp.js

```diff
@@ -0,0 +1,33 @@
+const days = [
```

Comment:
@jordanh sounds good, will do! Thought about that, just wasn't sure if it would even last in the design.